.github/workflows/build.yml

```diff
@@ -9,7 +9,15 @@ on:
 jobs:
   build:
     runs-on: ubuntu-latest
+    uses: [actions/checkout@v3, ./.github/actions/prepare]
     steps:
-      - uses: actions/checkout@v3
-      - uses: ./.github/actions/prepare
```

Comment:
OH!? `uses` can be an array? That is very nifty - would you be open to find-and-replacing _all_ the `github/workflows/*.yml` files in this repo in a separate PR? I like this 😄 thank you for showing it!

Reply:
I also need to test this out

Reply:
https://github.com/JoshuaKGoldberg/template-typescript-node-package/actions/runs/4396405450/workflow gives me the impression it's not actually allowed.

> You have an error in your yaml syntax on line 12

Reply:
@garuna-m6 I think `uses` isn't allowed to be an array. The `build` workflow isn't running. 😞

Reply:
Sorry about this, I remember reading somewhere we can use arrays and not able to locate the resource. Making changes against to fix it